### ISSUE TITLE
NPM updated

### ISSUE THREAD

**Issue by jasonmerry** [fritx/node-7zip#6]
Please can you publish your latest version to npm?

I am in need of your 'module not found' fix from 11 months ago in the 'index.js'.

Thanks in advance.

**Comment by fritx** [comment]
oh @jasonmerry , apology for missing information.

`7zip` on npm has been renamed to `win-7zip` (as it's only for win) since version 0.1.0
and commit https://github.com/fritx/win-7zip/commit/516898de1f34314adfbf524ba3aa335a5b82999c.

`7zip` would remain as 0.0.6.

**Comment by fritx** [comment]
One more thing, `cross-unzip` should be bumped meanwhile, (since version 0.0.3 and commit https://github.com/fritx/cross-unzip/commit/4203dd53f70ae27d1ddf29331234eb216ebbc187), to work together with `win-7zip`.

That is, with `win-7zip`, `cross-unzip` should be above 0.0.3 (latest is 0.2.1).

**Comment by jasonmerry** [comment]
Ah ha! lovely thanks.

might be worth mentioning that on npm? currently had over 10k downloads in the last week although possibly from dependencies. 

npm does however show your repo as being `win-7zip`

**Comment by fritx** [comment]
Yeah, sure, would add some migration guide, or any PR would be appreciated!
